style.css

```diff
@@ -151,10 +158,190 @@ body {
 }
 
 .element-right a {
-  position: static;
   width: 18px;
   height: 21px;
   left: 0;
   top: 9.35px;
   margin: 0 12px;
 }
+
+.work-container {
+  display: flex;
```

Comment:
- [ ] You should use CSS Grid layout for the works container as in the project requirement
![css-grid-flexbox](https://user-images.githubusercontent.com/44057863/166099939-4588e6f1-4de5-4109-a53b-d538d544d2d1.png)
.